cli/data.py

```diff
@@ -60,7 +60,7 @@ def update_forecasts(filename):
     "--aggregate-to-country/--no-aggregate-to-country",
     is_flag=True,
     help="Aggregate states to one USA country region",
-    default=True,
+    default=False,
```

Comment:
fwiw I added a line in the github actions yaml to disable county aggregations by default so not sure if this is necessary or not

Reply:
Yes, I saw that after making this change :-/ I like having `python ./run.py data update` reproduce what is in master.